src/modules/dmlink.js

```diff
@@ -2,17 +2,17 @@ const Eris = require("eris");
 const threadUtils = require("../threadUtils");
 
 /**
- * @param {Eris.CommandClient} bot 
+ * @param {Eris.CommandClient} bot
  */
 module.exports = bot => {
   threadUtils.addInboxServerCommand(bot, "dmlink", async (msg, args, thread) => {
-    if(! thread) return;
-    if (! args[0]) return "Provide a message ID";
+    if (! thread) return;
+    if (! args[0]) return msg.channel.createMessage("Please provide a message ID!");
     const [dmChannel, threadMessage] = await Promise.all([
-      bot.getDMChannel(thread.user_id),
+      thread.getDMChannel(),
       thread.getThreadMessageFromThread(args[0])
     ]);
-    if (! threadMessage) return msg.channel.createMessage("Message not found");
+    if (! threadMessage || threadMessage.message_type !== 3) return msg.channel.createMessage("Message not found!");
```

Comment:
`3` is the ID of messages sent from the user to the bot. Figured there wasn't any point sending the link for any other type of message since most of them (apart from messages sent *from* the bot) would be invalid. Can make it possible to view the link of messages with the ID of `4` if you think that's necessary.

Reply:
Yeah that would be good thank you

Reply:
Sick, done.